app/styles/ui/dialogs/_stash-changes.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../mixins';
+
+dialog#stash-changes {
+  width: 450px;
```

Comment:
Don't you still want to color the oction in the dialog?

```suggestion
  width: 450px;
  .dialog-content {
    .octicon {
      color: var(--dialog-warning-color);
      margin-right: var(--spacing);
    }
  }
```

Reply:
Hmm, I thought that was also baked in.